portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,25 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+//TODO: THIS! 
+ var pastSelection = null;
+ ; //just 
+
+ //e is the name of the option selection and name is the same as the id. 
+function changeData(e){ // function that the .html file will call: takes in the element the user selects from drop down menu and prints to console what was selected(passed into this function). The console can be viewed on the actual web page created: right click and then > select. 
+    console.log(e);
+        if(pastSelection != null){
```

Comment:
food for thought: != null covers both "null" and "undefined" cases. This is what we do in practice - why javascript has both null and undefined is kind of strange and I believe the original JS creator said it was a mistake. If you want to just guard against null alone, then use the `!==` operator instead. I think in this case, `!= null` is better since it covers both cases.